WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ReportList.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+import Yosemite
+
+protocol ReportListPresentable {
+    var items: [ReportItem] { get }
+    var title: String { get }
+    var ctaTitle: String { get }
+    var onDismiss: () -> Void { get }
+}
```

Comment:
I still do not understand the use-case for this protocol.

Reply:
It was meant to have a way to create viewModels to conform to it. For example, we had the WhatsNewViewModel that received a list of features and transformed into ReportItem. Now that the viewModel doesn't transform this data anymore, we might want to remove the need of this protocol! 
I will do it in a few 👍

Reply:
I've created this protocol so we can conform to it to have different title / ctaTitle, transform models into ReportItem, etc.
Now it is going to be used for WhatsNewViewModel only. Do you think we should stick without a boundary here and adopt something similar when needed in the future? (for the other reports to come)

Reply:
I would have probably renamed the view model to something like `ReportListViewModel` and let consumers create an instance. 

But this is also not wrong, so I'm fine if you choose to leave it like this!